catroid/src/main/java/org/catrobat/catroid/ui/fragment/ProjectOptionsFragment.kt

```diff
@@ -367,7 +367,7 @@ class ProjectOptionsFragment : Fragment() {
         project?.apply {
             val notificationData = StatusBarNotificationManager(context)
                 .createSaveProjectToExternalMemoryNotification(context, projectDestination, name)
-            ProjectExportTask(directory, projectDestination, notificationData, context).execute()
+            ProjectExporter(directory, projectDestination, notificationData, context)
```

Comment:
Where is the execution of the export method?

Reply:
Sorry I somehow forgot that I fixed it now